portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -344,6 +344,11 @@
 	<td>//div[(@class='form-group') and contains(@data-field-name,'${key_fieldName}')]//legend</td>
 	<td></td>
 </tr>
+<tr>
```

Comment:
Hi @LucasAcacio , 

If this path is only used on this test and you change the field type please remove it.

Thanks